apps/judicial-system/web/src/routes/Court/RestrictionCase/HearingArrangements/HearingArrangements.tsx

```diff
@@ -173,7 +173,7 @@ export const HearingArrangements = () => {
             <CourtArrangements
               handleCourtDateChange={handleCourtDateChange}
               handleCourtRoomChange={handleCourtRoomChange}
-              courtDate={courtDate}
+              courtDate={workingCase.arraignmentDate}
```

Comment:
Please reconsider this change, it breaks the functionality of [using the requestedCourtDate the first time court user lands on the page](https://github.com/island-is/island.is/blob/1a09eda74a257821fa37c19ce8f449fb4e53c366/apps/judicial-system/web/src/routes/Court/RestrictionCase/HearingArrangements/HearingArrangements.tsx#L62).

Reply:
Goot catch. I have fixed this on the hearing arrangement pages for both restriction cases and investigation cases.